apps/testing/src/main/java/com/akto/testing/Main.java

```diff
@@ -196,6 +197,42 @@ public static void main(String[] args) throws InterruptedException {
                         Filters.eq("_id", testingRun.getId()),  completedUpdate
                 );
 
+                if(summaryId != null){
+                    TestingRunResultSummary testingRunResultSummary = TestingRunResultSummariesDao.instance.findOne(
+                        Filters.eq(TestingRunResultSummary.ID, summaryId));
+                    if(testingRunResultSummary.getState() == TestingRun.State.RUNNING
+                            || testingRunResultSummary.getState() == TestingRun.State.SCHEDULED){
+
```

Comment:
why are we recalculating all this(COUNT_ISSUES etc), if it's already being done inside apiwiseinit

Reply:
let's not do this in this way. Let's identify what are the possible reasons why status is not getting updated and handle in properly. No additions in main.java

Reply:
Sure, let me refractor this PR to only add error messages for failing test runs.